client/src/components/game/star/StarDetail.vue

```diff
@@ -28,14 +28,18 @@
           <p class="mb-0" v-if="star.isAsteroidField">This star is surrounded by an <span class="text-warning">Asteroid Field</span>.</p>
           <p class="mb-2 text-info" v-if="star.isAsteroidField"><small><i>Asteroid Fields start with additional natural resources and x2 Defender Bonus (net +2 Weapons).</i></small></p>
         </div>
-        
+
         <div v-if="star.wormHoleToStarId">
           <hr/>
           <p class="mb-0" v-if="wormHolePairStar">This star is a <strong>Worm Hole</strong> to <a href="javascript:;" @click="viewOnMap(wormHolePairStar)"><i class="fas fa-eye mr-1"></i>{{wormHolePairStar.name}}</a>.</p>
           <p class="mb-0" v-if="!wormHolePairStar">This star is a <strong>Worm Hole</strong> to an unknown star.</p>
           <p class="mb-2 text-info"><small><i>Travel between Worm Holes takes 1 tick.</i></small></p>
         </div>
-
+        <div v-if="star.warpGate">
+        <hr/>
+          <p class="mb-0">This star has a <strong>Warp Gate</strong><i class="fas fa-dungeon ml-2"></i>.</p>
+          <p class="mb-2 text-info"><small><i>Carriers travel {{ warpSpeedMultiplier }}x faster between warp gates.</i></small></p>
```

Comment:
"between active warp gates"

Warp gates at abandoned stars are inactive so we should make that distinction 👍